pending/tg.js

```diff
@@ -0,0 +1,749 @@
+module.exports = `
+Menu.btn.ranked niranggo
+menu.btn.shop Tindahan
+menu.btn.social panlipunan 
+menu.btn.maps Laro
+menu.btn.mods Mods
+menu.btn.settings Settings
+menu.btn.customize Ipasadya
+menu.btn.invite Mag-anyaya
+menu.btn.invite.clicked Nakopya
+menu.btn.join Sumali
+menu.btn.host Host Game ng Host
+menu.btn.browser Mga Server
+generic.wait hintayin...
+generic.loading LOADING...
+generic.none Wala
+generic.connecting KUMOKONEKTA...
+generic.select Pumili
+generic.delete Tanggalin
+generic.save Magtipid
+generic.load Mag-Load
+generic.search Maghanap
+generic.leave Umalis
+generic.default Default
+generic.submit Ipasa
+generic.create CreatLumikha
+generic.random Random
+generic.failed Nabigo
+generic.success Tagumpay!
+generic.name Pangalan
+generic.level Antas
+generic.score Iskor
+generic.off Off
+generic.defeat PAGKATALO
+generic.victory TAGUMPAY /Panalo?
```

Comment:
TAGUMPAY is more appropriate, Panalo is considered "win" but not necessarily victory